openmetadata-ui/src/main/resources/ui/src/rest/databaseAPI.ts

```diff
@@ -86,6 +88,59 @@ export const patchDatabaseSchemaDetails = async (
   return response.data;
 };
 
+export const addSchemaFollower = async (id: string, userId: string) => {
+  const response = await APIClient.put<
+    string,
+    AxiosResponse<{
+      changeDescription: { fieldsAdded: { newValue: EntityReference[] }[] };
+    }>
+  >(
+    `databaseSchemas/${id}/followers`,
+    userId,
+    APPLICATION_JSON_CONTENT_TYPE_HEADER
+  );
+
+  return response.data;
+};
+
+export const addDatabaseFollower = async (id: string, userId: string) => {
+  const response = await APIClient.put<
+    string,
+    AxiosResponse<{
+      changeDescription: { fieldsAdded: { newValue: EntityReference[] }[] };
+    }>
+  >(`databases/${id}/followers`, userId, APPLICATION_JSON_CONTENT_TYPE_HEADER);
+
+  return response.data;
+};
+
+export const removeSchemaFollower = async (id: string, userId: string) => {
+  const response = await APIClient.delete<
+    string,
+    AxiosResponse<{
+      changeDescription: { fieldsDeleted: { oldValue: EntityReference[] }[] };
+    }>
+  >(
+    `/databaseSchemas/${id}/followers/${userId}`,
+    APPLICATION_JSON_CONTENT_TYPE_HEADER
+  );
+
```

Comment:
optimise this, not in favor of adding addFollower and removeFollower for every entity